src/libcore/ops.rs

```diff
@@ -95,6 +95,16 @@ use fmt;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait Drop {
     /// A method called when the value goes out of scope.
+    ///
+    /// When this method has been called, `self` has not yet been deallocated.
```

Comment:
I think the tense in the first clause makes things a bit ambiguous. How about something like "Before this method is called, `self` has not yet been deallocated."